tests/server_unit_test/test_admin.py

```diff
@@ -134,3 +134,43 @@ def test_add_file_no_store_exists(test_client):
     response = AdminRequestFailedResponse.model_validate_json(response.content)
 
     assert response.reason == "Store not_a_store does not exist."
+
+def test_verify_file_success(test_client, test_server, garbage_file, test_orm):
+    """
+    Tests that a file's properties match the database record.
+    """
+    setup = test_server[2]
+    store = setup.store_directory
+    full_path = store / "test_file_to_verify.txt"
+    # Create the file in the store
+    shutil.copy2(garbage_file, full_path)
+
+    # Assume the file has been added to the database already; here we simulate the verification request
+    request = {
+        "name": "test_file_to_verify.txt",
+        "size": get_size_from_path(full_path),
+        "checksum": get_md5_from_path(full_path),
+        "store_name": "local_store",
+    }
```

Comment:
It's good to exercise the pydantic request model here, instead of writing the JSON yourself.